packages/react/src/ReactLazy.js

```diff
@@ -49,44 +49,62 @@ export type LazyComponent<T, P> = {
 };
 
 function lazyInitializer<T>(payload: Payload<T>): T {
+  function onError(error) {
+    if (payload._status === Pending) {
+      // Transition to the next state.
+      const rejected: RejectedPayload = (payload: any);
+      rejected._status = Rejected;
+      rejected._result = error;
+    }
+  }
+
   if (payload._status === Uninitialized) {
     const ctor = payload._result;
     const thenable = ctor();
     // Transition to the next state.
     const pending: PendingPayload = (payload: any);
     pending._status = Pending;
     pending._result = thenable;
-    thenable.then(
-      moduleObject => {
-        if (payload._status === Pending) {
-          const defaultExport = moduleObject.default;
-          if (__DEV__) {
-            if (defaultExport === undefined) {
-              console.error(
-                'lazy: Expected the result of a dynamic import() call. ' +
-                  'Instead received: %s\n\nYour code should look like: \n  ' +
-                  // Break up imports to avoid accidentally parsing them as dependencies.
-                  'const MyComponent = lazy(() => imp' +
-                  "ort('./MyComponent'))",
-                moduleObject,
-              );
-            }
+    thenable.then(moduleObject => {
+      if (payload._status === Pending) {
+        let defaultExport;
+        if (__DEV__) {
+          if (moduleObject === undefined) {
+            console.error(
+              'lazy: Expected the result of a dynamic import() call. ' +
+                'Instead received: %s\n\nYour code should look like: \n  ' +
+                // Break up imports to avoid accidentally parsing them as dependencies.
+                'const MyComponent = lazy(() => imp' +
+                "ort('./MyComponent'))\n\n" +
+                'Did you accidentally put curly braces around the import?',
+              moduleObject,
```

Comment:
Just curious: What's the value of passing `moduleObject` to the console here, given that it's `undefined`

Reply:
Just copypaste from below, no particular difference unless someone later changes the condition.

Reply:
I can hardcode it.